CODEOWNERS

```diff
@@ -6,82 +6,82 @@
 #########
 
 # PRLabel: %Cognitive Services
-/dev/cognitiveservices/data-plane/Language/ @assafi @rokulka @ChongTang @annatisch @heaths @Azure/api-stewardship-board
+/dev/cognitiveservices/data-plane/Language/ @assafi @ChongTang @annatisch @heaths @Azure/api-stewardship-board
 
-/specification/adp/ @Azure/adp
+/specification/adp/
```

Comment:
It was my understanding that the automation needs at least one id following the directory name to work correctly, so removing the last id for a directory seems problematic.